test/test_custom_ops.py

```diff
@@ -865,6 +865,14 @@ def foo(x: Tensor, y: Tuple[int, int]) -> Tensor:
 
             del foo
 
+        with self.assertRaisesRegex(ValueError, r"List\[int\]"):
```

Comment:
Can you include the "For example, " in the regex? Just to check that this is human readable (and not a dump of all of the types like it was before)